ImageFeed/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+final class ProfileViewController: UIViewController{
+    @IBOutlet private var avatarImageView: UIImageView!
```

Comment:
@IBOutlet - не нужны здесь, ведь верстаешь кодом:
```suggestion
    private var avatarImageView = UIImageView()
```

Reply:
а потом увидела что вообще эти свойства не используются, а инициализация во viewDidLoad

Обычно хорошо бы держать ссылку на вьюху, для легкого доступа. А так получается ссылки нет (вью содержит вьюшку, а в коде ссылка будет утерена – обновить поля не получится динамически. 

Поэтому давай инициализацию сделаем здесь, а во вьюдидлоад только настройку оставим